sound/soc/sof/intel/cnl.c

```diff
@@ -216,6 +216,7 @@ const struct snd_sof_dsp_ops sof_cnl_ops = {
 	.debug_map_count	= ARRAY_SIZE(cnl_dsp_debugfs),
 	.dbg_dump	= hda_dsp_dump,
 	.ipc_dump	= cnl_ipc_dump,
+	.dbg_init	= hda_dsp_dbg_init,
```

Comment:
we already have a core initialization of debugfs, so that is not needed?

Reply:
As I did not know how to abstract probes without HDA, I did a simple trick - abstracted dbg init instead and so there is no code generated for non-Intel uses and plenty added for Intel side. Everyone is happy probes worked : )

Reply:
don't get the answer, sorry